lib/support/preURL.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const Promise = require('bluebird'),
+  log = require('intel');
+
+module.exports = {
+  run(context) {
+      return context.runWithDriver((driver) => {
+          log.info('Accessing preURL %s', context.options.preURL);
+          return driver.get(context.options.preURL);
+        }).then(() => Promise.delay(1500));
```

Comment:
yes that could probably do that! we still need that delay after the orange page though.